OHBM_hackathon_2019_project_description.md

```diff
@@ -0,0 +1,74 @@
+
+So the short term goal of this project is to facilitate the use of this checklist. But, if done right, this could also in the long term enhance the adoption of emerging neuroimaging standards ([the Brain imaging data structure](https://github.com/ohbm/hackathon2019/blob/master/Tutorial_Resources.md#the-brain-imaging-data-structure-bids), fMRIprep, NIDM...), facilitate data sharing and pre-registration, help with peer-review...
+
+### Short term goals
+
+**The short term goal of this project is to make the Appendix D checklist easier to use**.
+
+One way to achieve this may be to turn the checklist into a website that users can click through and provide the information requested by COBIDAS. Ideally this would generate a small text file (for example a json file) that summarizes what option was chosen for each item of the checklist. This machine readable file could then be used to automatically generate part of the methods section of an article.
+
+A side branch of this project includes developing toolboxes  for the main neuroimaging software (SPM, FSL...) that given a processing batch file as input (e.g matlabbatch.mat for SPM or design.fsf for FSL) creates an output json file readable by the COBIDAS checklist website.
+
+Other potential goals (interaction with BIDS and NIDM, further integration with main neuroimaging softwares) and potential applications (improving data-sharing and peer-review) of this project are described in this [repository](https://github.com/Remi-Gau/COBIDAS_chckls).
+
+#### Requirements
+
+The implementation of this project should remain flexible enough to:
+- accommodate the inclusion of new items in the checklist as new neuroimaging methods mature (e.g new multivariate analysis, high-resolution MRI...),
+- easily fork the project and convert it to create a checklist-website for a different field. In practice, this will most likely involve mean a deployment through a container technology like [docker](https://github.com/ohbm/hackathon2019/blob/master/Tutorial_Resources.md#containers).
+
+
+## Skills required to participate
+
+- To be enthusiastic about reproducibility
+- Familiarity with the COBIDAS report for f/MRI, MEEG (not necessary)
+- To know about web design (not necessary)
+- Familiarity with one or more of the main neuroimaging software for [fMRI](https://github.com/ohbm/hackathon2019/blob/master/Tutorial_Resources.md#neuroimaging) (SPM, FSL...) or for [M/EEG](https://github.com/ohbm/hackathon2019/blob/master/Tutorial_Resources.md#main-eeg-and-meg-softwares) (Fieldtrip, EEGlab...) (not necessary)
+
+
+## Milestones
+
+- Discuss conceptual and structural details of the COBIDAS-json file.
+
+- Create a template of the COBIDAS-json file
+
+- Create a proof of concept website that
+  - generates a checklist to by users from an empty template COBIDAS-json file,
```

Comment:
This sentence doesn't make sense, but I'm not sure what to change in order to improve it. @Remi-Gau maybe rephrase to highlight the intended meaning?